### DISCUSSION TITLE
Allow CarbonImmutable for CarbonPayload

### DISCUSSION THREAD

**Discussion by abenerd** [spatie/ray#315] [Ideas]
In my Laravel projects I prefer `CarbonImmutable` over `Carbon`, therefore I propose that when instantiating the `\Spatie\Ray\Payloads\CarbonPayload` to allow an instance of `CarbonInterface` to be the argument rather than the more specific `Carbon` class. This would allow people to use both `Carbon` and `CarbonImmutable`.

**Comment by freekmurze** [comment]
Good idea, please send a PR for this. 

If no PR comes in, I'll take care of it soon.

**Reply by freekmurze** [reply]
Thanks for the PR, merged!

**Reply by abenerd** [reply]
Great, thanks!
